src/index.ts

```diff
@@ -632,6 +664,16 @@ class Piscina extends EventEmitter {
     return this.#pool.taskQueue.length;
   }
 
+  get queueLoad () : number {
+    // If maxQueue is Infinity, queueLoad is always either 1 or 0,
+    // otherrwise, queueLoad is the ratio of queueSize / maxQueue
+    if (this.#pool.options.maxQueue === Number.POSITIVE_INFINITY) {
+      return this.queueSize > 0 ? 1 : 0;
+    } else {
+      return this.queueSize / this.#pool.options.maxQueue;
```

Comment:
`this.#pool.options.maxQueue` can be `0`, btw

Reply:
Ah, right right. :-)